.goreleaser.yaml

```diff
@@ -26,9 +26,12 @@ builds:
     main: cli/main.go
 
 archives:
-  - format: binary
+  - format: tar.gz
     name_template: "{{ .Binary }}_{{ .Version }}"
```

Comment:
I noticed after I added this that this is `_` instead of `-`, can you correct it here?

Reply:
I think we should stick the `v` prefix back in also, not sure why goreleaser strips that for archives